File: CS50x/Week1_C/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // ask for the height
    int height;
    do
    {
        height = get_int("give me the peramide height : ");
    }
    while (height < 1 || height > 8);
    // peramide
    int dots = height - 1;
    int sharps = height - dots;
    for (int i = 0 ; i < height; i++)
    {

        for (int j = 0 ; j < dots ; j++)
        {
            printf(" ");
        }
        for (int h = 0 ; h < sharps ; h++)
        {
            printf("#");
        }
        printf("\n");
        dots --;
        sharps++;
    }

}

File: CS50x/Week6_Python/sentimental-mario-less/mario.py
# TODO
while True:
    height = input("Give me the pyramid height (1-8): ")
    if height.isdigit():
        height = int(height)
        if 1 <= height <= 8:
            break


dots = height - 1
sharps = height - dots
for i in range(height):
    for j in range(dots):
        print(" ", end="")
    for k in range(sharps):
        print("#", end="")
    print()
    dots = dots - 1
    sharps = 1 + sharps
